cloudferrylib/os/storage/cinder_database.py

```diff
@@ -29,7 +29,7 @@
 HOST = 'host'
 
 
-class CinderStorage(storage.Storage):
+class CinderStorage(cinder_storage.CinderStorage):
```

Comment:
At least it works for me =)
I plan rename it when will do plugin system implementation.